tools/java/org/unicode/cldr/tool/ChartChurn.java

```diff
@@ -0,0 +1,1116 @@
+package org.unicode.cldr.tool;
```

Comment:
This is doing a lot of unnecessary work, I think because it was copied from the delta chart; that chart is much more complicated because it is looking at changes in various attributes, etc. and also looking at more than just main+annotations. We don't need to bother with the non-ldml pages, for example.

It might be the wrong format also, because we want to have one big chart that we can review, including all of the locales in one chart instead of being split into hundreds of locales. More like the Flagged chart in that respect. (You might have changed that but hard to tell because so much else is going on.)